name: "Bob"
drift_value: 0.2
delay_time: 3.0
